apps/next/src/components/ui/card/food-card.tsx

```diff
@@ -32,31 +36,58 @@ const FoodCardContent = React.forwardRef<
 >(({ dish, ...divProps }, ref) => {
   const IconComponent = getFoodIcon(dish.name) ?? Utensils;
 
-  const caloricInformationAvailable: boolean = dish.nutritionInfo.calories != null
-    && dish.nutritionInfo.calories.length > 0;
+  const utils = trpc.useUtils();
+  const logMealMutation = trpc.nutrition.logMeal.useMutation({
+    onSuccess: () => {
+      alert(`Added ${formatFoodName(dish.name)} to your log`);
```

Comment:
We will eventually want to make this into a [toast/sonner](https://ui.shadcn.com/docs/components/sonner). Adding a comment here for a reminder would be good:

```suggestion
      //TODO: Replace this with a shad/cn sonner or equivalent.
      alert(`Added ${formatFoodName(dish.name)} to your log`);
```